src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
It would be a BC break yes. What about type hinting against `HtmlDumper`? I don't think it makes  much sense to allow the DumpExtension to output anything but HTML. This is not perfect in OOP design, but hey, this is a twig extension (i.e. glue code), not some architectural center piece. WDYT?